### PR TITLE
Pt september 2016

### PR DESCRIPTION
**Pull request by llvasconcellos**
Hi, I fixed indentation and added translation for the new strings.

### CODE REVIEW

File: t9n/pt.coffee [hunk 1]
```diff
@@ -42,16 +42,30 @@ pt =
   username: "Nome de usuário"
   usernameOrEmail: "Usuário ou e-mail"
   with: "com",
-  "Send the verification email again" : "Reenviar e-mail de verificação",
```
Comment (softwarerero) [inline t9n/pt.coffee]: Why did you delete these?

File: t9n/pt.coffee [hunk 2]
```diff
@@ -42,16 +42,30 @@ pt =
...
-  "Send email again" : "Reenviar e-mail"
```
Comment (softwarerero) [inline t9n/pt.coffee]: Why did you delete this?

File: t9n/pt.coffee [hunk 3]
```diff
@@ -67,54 +81,59 @@ pt =
 
 
     accounts:
```
Comment (softwarerero) [inline t9n/pt.coffee]: It looks like this should be indented under accounts.

File: t9n/pt.coffee (hunk 2 above)
Comment (llvasconcellos) [inline t9n/pt.coffee]: I did a diff with the english version and both wheren't there. I noticed the key value for resendVerificationEmail and I added it. Also both are the quite the same message.

File: t9n/pt.coffee (hunk 3 above)
Comment (llvasconcellos) [inline t9n/pt.coffee]: You are right! My bad! Done!
